packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/check_reference_origins.ts

```diff
@@ -43,7 +43,6 @@ async function checkOrigin(
     rootSearchFields: ['_id', 'originId'],
     page: 1,
     perPage: 1, // we only need one result for now
-    fields: ['title'], // we don't actually need the object's title, we just specify one field so we don't fetch *all* fields
```

Comment:
We may want to add an option for retrieving the root fields only

Reply:
Perhaps an option to return just "meta" fields like ID, createdAt, updatedAt?